### ISSUE TITLE
Content Writing

### ISSUE THREAD

**Issue by machuga** [coderie/coderie-site#2]
The home page and an about page will need some content about who we are, what we are here for, and any extra information about the group.  We need someone to start writing out this content.  If you submit pages via a PR we can all help you review and edit the content.

Any takers?

**Comment by saraolives** [comment]
I'm in!

**Comment by matsilva** [comment]
... initial content ideas from the entire group could be potentially helpful for whomever takes a stab at it

**Comment by machuga** [comment]
@matsilva yup! Let's utilize GitHub to its fullest for collaboration.

**Comment by lxcodes** [comment]
@saraolives Did you have a place where you were working on drafts?

**Comment by saraolives** [comment]
No, Matt mentioned just putting the drafts up here. Maybe we could have a google doc too, in case anyone wants to do some real-time collaboration?

**Comment by Bshally** [comment]
Google docs is great in my opinion so that we can have the initial content creation happen there with edits marked by person (google's suggested edit functionality is pretty slick), and all that then move the results to github and maintain them from pull requests from there once they are incorporated with the page designs . Just my 3 cents.

**Comment by lxcodes** [comment]
I'm down with the Sara and Brandon.

**Comment by thumper34** [comment]
Feel free to drop me the Google Docs link and I will take a look :)

**Comment by lxcodes** [comment]
For those not in Slack: https://drive.google.com/open?id=0B5-EnF2L4i1nfmd1OHVCSmRUclJqcEpXdFY5UEJwVmpYMEdjTnFhbEs0RnluSVJ3emdHWlU
